src/net.h

```diff
@@ -697,6 +711,22 @@ class CNode
     static uint64_t GetTotalBytesRecv();
     static uint64_t GetTotalBytesSent();
 
+    void AccountForSentBytes(const std::string& msg_type, size_t sent_bytes)
+        EXCLUSIVE_LOCKS_REQUIRED(cs_vSend)
+    {
+        mapSendBytesPerMsgType[msg_type] += sent_bytes;
+    }
+
+    void AccountForRecvBytes(const std::string& msg_type, size_t recv_bytes)
+        EXCLUSIVE_LOCKS_REQUIRED(cs_vRecvMsg)
+    {
+        auto msgTypeInMap = mapRecvBytesPerMsgType.find(msg_type);
+        if (msgTypeInMap == mapRecvBytesPerMsgType.end())
+            msgTypeInMap = mapRecvBytesPerMsgType.find("*other*");
```

Comment:
Can we use the message type instead of the string value itself?

```suggestion
            msgTypeInMap = mapRecvBytesPerMsgType.find(NetMsgType::OTHER);
```

Reply:
Yes, that's a better option, thanks!